salt/metalk8s/kubeadm/init/kubeconfig/lib.sls

```diff
@@ -0,0 +1,15 @@
+{%- macro kubeconfig(name, conf) %}
+Create kubeconf file for {{ name }}:
+  metalk8s_kubeconfig.managed:
+    - name: {{ conf.file }}
+    - client_cert_info:
+        signing_private_key: /etc/kubernetes/pki/ca.key
+        signing_cert: /etc/kubernetes/pki/ca.crt
+        {%- for key, value in conf.cert.items() %}
+        {{ key }}: {{ value }}
+        {%- endfor %}
+    - config_context:
+        kube_apiserver: https://10.200.5.53:6443
```

Comment:
Will update the api server address

Reply:
Suggest something like `https://{{ salt['network.ip_addrs'](cidr=network.control_plane)[0]:6443` (network.control_plane is part of the pillar in my PR)